hw/top_earlgrey/data/ip/chip_clkmgr_testplan.hjson

```diff
@@ -187,8 +187,10 @@
             jitter_o clkmgr output to toggle. Verify this output is connected to AST's
             clk_src_sys_jen_i input using formal.
 
+            If jitter_regwen is enabled check that jitter_enable can be toggled.
+            If jitter_regwen is disabled check that jitter_enable cannot be changed.
```

Comment:
Nit: the test always disables the jitter_regwen and then checks that jitter_enable cannot be changed. I think this is great. The test can run independent whether the jitter_enable is locked already.

Reply:
The test depends on the current state of jitter_regwen: if it is unlocked it checks that jitter_enable can be flipped, and if locked it checks it cannot. This is per line 50 in clkmgr_jitter_test.c.